src/main/java/uk/gov/hmcts/dts/fact/entity/Contact.java

```diff
@@ -23,23 +23,21 @@
     private String explanationCy;
     private boolean inLeaflet;
     private boolean fax;
-    private Integer sortOrder;
 
     @OneToOne()
     @JoinColumn(name = "contact_type_id")
     private ContactType contactType;
 
     public Contact(final ContactType contactType, final String number, final String explanation, final String explanationCy) {
-        this(contactType, number, explanation, explanationCy, false, null);
+        this(contactType, number, explanation, explanationCy, false);
```

Comment:
Food for thought on this one, do any functional tests on the frontend or admin portal rely upon the sort-order being set? Might be worth checking they all pass too when used alongside this code